tests/test_unit.py

```diff
@@ -442,30 +442,25 @@ def test_get_project_success(client):
 
 def test_get_project_returns_404_if_unknown_project_hash(client):
     hash = "unit_test_not_known"
-    result = app.get_project(hash)
-    assert result[1] == 404
-    response = read_data(result[0])["error"]
-    data = response["data"]
-    errors = response["errors"]
-    assert errors[0] == {
-        "error": "Project hash not found",
-        "detail": "Project hash does not have an associated job",
-    }
+    with pytest.raises(NotFound) as e_info:
+        app.get_project(hash)
+    assert (
+        e_info.value.description
+        == "Project hash does not have an associated job"
+    )
 
 
 @patch("rq.job.Job.fetch")
-def test_get_project_returns_500_if_status_error(mock_fetch):
+def test_get_project_returns_404_if_status_error(mock_fetch):
```

Comment:
```suggestion
def test_get_project_returns_404_if_project_not_found(mock_fetch):
```
Presumably it would return a 500 if there was a server error for some reason,